### PR TITLE
[fix] Allow interactions with embeds in readonly mode

### PR DESCRIPTION
**Pull request by MitjaBezensek**
Allow the users to interact with embeds in readonly mode. Not sure if this should apply to all embeds (cc @orangemug for thoughts on this)? For example, you can also start editing code sandbox embeds. 

One thing that doesn't feel quite right is that readonly mode defaults to the hand tool, so you always have to switch to select tool to get this working. I guess moving around is still the more common action though so 🤷 

### Test Plan

1. Create a multiplayer room with some embeds (youtube videos, spotify playlist).
2. Open the room in readonly mode.
3. Make sure you can interact with embeds. Double click / enter when selected, you should then be able to play the youtube videos.

- [x] Unit Tests
- [ ] Webdriver tests

### Release Note

- Allow the users to interact with embeds in readonly mode.

### CODE REVIEW

File: packages/editor/src/lib/app/statechart/TLSelectTool/children/Idle.ts [hunk 1]
```diff
@@ -267,43 +267,71 @@ export class Idle extends StateNode {
...
-		switch (info.code) {
-			case 'Enter': {
-				const { selectedShapes } = this.app
-
-				if (selectedShapes.every((shape) => shape.type === 'group')) {
-					this.app.setSelectedIds(
-						selectedShapes.flatMap((shape) => this.app.getSortedChildIds(shape.id))
-					)
-					return
+		if (this.app.isReadOnly) {
```
Comment (MitjaBezensek) [inline packages/editor/src/lib/app/statechart/TLSelectTool/children/Idle.ts:270]: Felt better to create a separate branch for readonly and only handle the single scenario that applies to that mode.

**Comment by orangemug** [comment]
> Not sure if this should apply to all embeds (cc @orangemug for thoughts on this)? For example, you can also start editing code sandbox embeds.

Sorry didn't see this one. Hmmm I'm not really sure, I mean embeds are just content stored externally. You can edit the content remotely and refresh the page and it'd update. But also it does seem odd to be modifying content in a read only room.

I think we should allow all embeds to be interactive, this is because making codesandbox non-interactive (for an example) would also mean you couldn't open the links to non-embedded codesandbox.

**Comment by MitjaBezensek** [comment]
Sounds good, so we can leave it as is.

**Comment by steveruizok** [comment]
Looks good!

**Pull request merged**